src/Features/CSharp/Portable/ConvertLinq/ConvertForEachToLinqQuery/SyntaxNodeAndSyntaxTokenExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.ConvertLinq.ConvertForEachToLinqQuery;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.ConvertLinq.ConvertForEachToLinqQuery
+{
+    internal static class SyntaxNodeAndSyntaxTokenExtensions
```

Comment:
can we make this generic helper. this is nice!

Reply:
I think we have workspace shared folder or something where we put these kind of helpers.